doc/CHEATSHEET.md

```diff
@@ -487,7 +480,7 @@ foo :: MonadBlockChain m => m ()
 foo = do
     bar `withTweak` ( do
                         C.overTweak
-                          (txSkelOutsL % _head % txSkelOutValueL) -- Value of first output
+                          (txSkelOutsL % ix 1 % txSkelOutValueL) -- Value of first output
```

Comment:
Not very knowledgeable about the optics library: why the change? Is `ix` for indexes in both lists and tuples whereas `_head` is just for lists?

Reply:
`ix` is general to indexed optics and `_head` is specific to lists. I changed it because I wanted the example to be as generic as possible (open the possible to `ix n`) while `_head` can literally just be used for the first element.